install/installer/pkg/components/slowserver/deployment.go

```diff
@@ -323,6 +323,14 @@ func deployment(ctx *common.RenderContext) ([]runtime.Object, error) {
 		return nil
 	})
 
+	dbWaiterDbEnv := common.DatabaseEnv(&ctx.Config)
+	for i := range dbWaiterDbEnv {
+		if dbWaiterDbEnv[i].Name == "DB_HOST" {
+			dbWaiterDbEnv[i].ValueFrom = nil
+			dbWaiterDbEnv[i].Value = toxiproxy.Component
+		}
+	}
```

Comment:
This feels awkward, but considering we'd change `common.DatabaseEnv` to have this logic feels even worse, as the function no longer has context of the slow-server deployment.

Reply:
I agree, but seeing as this component is temporary, generalising the `common.DatabaseEnv` function is probably the wrong way to go here.